zfs_test/replicate_test/task_test/report_test.py

```diff
@@ -0,0 +1,21 @@
+"""zfs.replicate.task.report tests."""
```

Comment:
Unrelated file change?

Reply:
Added to keep test coverage happy. Apparently the coding lines provided noticeable covered lines. :p